mails/_partials/download_product_virtual_products.tpl

```diff
@@ -24,6 +24,6 @@
  *}
 <ul>
 {foreach $list as $product}
-  <li><a href="{$product.url}">{$product.text}</a></li>
+  <li><a href="{$product.url}">{$product.text}</a>{$product.complementary_text}</li>
```

Comment:
It should be better to add a trim for removing space at start.

Reply:
I'm not sure because the space is added on purpose :

`$complementaryText .= ' ' . $this->trans('expires on %s.', array($product['deadline']), `

Reply:
Ok; Sorry.